src/mpyl/steps/deploy/dagster.py

```diff
@@ -33,6 +33,7 @@
 
 
 class DagsterBase:
+    dagster_helm_chart_version = "1.9.5"
```

Comment:
Maybe this value should come from the configuration (in the scripting repo)?

Reply:
It's the nicer approach yes, but it's more work (I did it anyway, I just want to whine)